extensions/amp-auto-ads/0.1/attributes.js

```diff
@@ -29,19 +29,31 @@ const NON_DATA_ATTRIBUTE_WHITELIST = {
   'type': true,
 };
 
+/**
+ * Indicates attributes from config object for different ad formats.
+ * @enum {string}
+ */
+export const Attributes = {
+  // Attributes from config object which should be added on any ads.
+  BASE_ATTRIBUTES: 'attributes',
+  // Attributes from config object which should be added on anchor ads.
+  STICKY_AD_ATTRIBUTES: 'stickyAdAttributes',
```

Comment:
At some point you should update the MD file with this new API field: https://github.com/ampproject/amphtml/blob/master/extensions/amp-auto-ads/amp-auto-ads.md#configobj

Reply:
Updated MD file.